.github/workflows/cxx-build.yml

```diff
@@ -79,24 +90,12 @@ jobs:
           name: core
           path: src/sdks/core
 
-      - name: Upload Manage SDK
-        uses: actions/upload-artifact@v3
-        with:
-          name: manage
-          path: src/sdks/manage
-
-      - name: Upload Discovery SDK
-        uses: actions/upload-artifact@v3
-        with:
-          name: discovery
-          path: src/sdks/discovery        
-
   core_sdk:
     name: Build Core SDK
     needs: [thunder, openrpc]
     runs-on: ubuntu-latest
     container:
-      image: kevinshahfws/node-c:3.0
+      image: node:latest
```

Comment:
Can we manually specify whatever the latest version is here? It wouldn't be the first time that "latest" changed to something that broke a pipeline.

Reply:
fixed